shopinvader_api_quotation/routers/cart.py

```diff
@@ -0,0 +1,37 @@
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import Depends
+
+from odoo import api, models
+
+from odoo.addons.base.models.res_partner import Partner as ResPartner
+from odoo.addons.fastapi.dependencies import (
+    authenticated_partner,
+    authenticated_partner_env,
+)
+from odoo.addons.shopinvader_api_cart.routers import cart_router
+from odoo.addons.shopinvader_schema_sale.schemas.sale import Sale
+
+
+@cart_router.post("/{uuid}/request_quotation")
```

Comment:
To not break extension mechanism from odoo, you must define a new router and explain into your readme that your new router must be mounted at the same point as the initial cart router as I did it for the `shopinvader_api_sale_loyalty` addon in https://github.com/shopinvader/odoo-shopinvader/pull/1558/commits/2c20ea840bdd15190d1466cd96eeee0cb3d81416